src/extensions/offChainTrades.se

```diff
@@ -32,31 +32,26 @@ event Trade(maker:address:indexed, taker:address, tokenX:address:indexed, tokenY
 def init():
     self.controller = 0x0
 
-def trade(maker: address, v, r, s, tokenX: address, tokenY: address, orderSize, takerAmount, fxpPrice, expiration, gasPriceLimit):
+def publicTrade(maker: address, v, r, s, tokenX: address, tokenY: address, orderSize, takerAmount, fxpPrice, expiration, gasPriceLimit):
```

Comment:
There are a bunch of these, they are all just to get the contracts compiling and tests passing as these files will be deleted in reporting branch.  I wouldn't look too deeply into them beyond making sure they are in a file that is in fact going to be deleted.